generated/nidaqmx/_task_modules/channels/ai_channel.py

```diff
@@ -3579,7 +3578,7 @@ def ai_dither_enable(self):
         check_for_error(error_code)
 
     @property
-    def ai_eddy_current_prox_sensitivity(self):
+    def ai_eddy_current_prox_probe_sensitivity(self):
```

Comment:
The old short names matches the LV short name. The new short name matches the C name. So there isn't a right answer here. I prefer this new name, but I really like the deprecation idea @bkeryan suggested, so let's add the old one as deprecated.